starboard/linux/shared/platform_configuration/BUILD.gn

```diff
@@ -34,7 +34,7 @@ config("compiler_flags") {
       "-fcolor-diagnostics",
 
       # Default visibility to hidden, to enable dead stripping.
-      "-fvisibility=hidden",
+      "-fvisibility=default",
```

Comment:
@y4vor to confirm : changing visiiblity to DEFAULT for starboard symbols makes sense in all configs : monolithic,  modular and evergreen.

Starboard symbols should always be default visibility IMO.

Althought im not sure how this worked in evergreen when the starboard symbol visibility was hidden.

Reply:
I think we should keep them hidden by default and only for the modular build export only the Sb* symbols. 
There is a runtime performance implications when too many symbols are public. 
It could be a separate CL as this one is quite big already.

We can try something with: starboard/starboard.syms
starboard_syms_path =   rebase_path("//starboard/starboard.syms", root_build_dir)
    ldflags = [
      "-Wl,--dynamic-list=$starboard_syms_path",
    ]